### ISSUE TITLE
Activities - Global Dashboard

### ISSUE THREAD

**Issue by yabellini** [rladies/website#229]
This option in the menu link to https://benubah.github.io/r-community-explorer/rladies.html and this dashboard is not updated, as we already have all the info in our webpage I think we can retire this link.  As it is outdated it leads to confusion and we do not control that dashboard.

**Comment by cosimameyer** [comment]
I agree - I think it might be misleading if the data is not continuously updated.

**Comment by drmowinckels** [comment]
we have removed the dashboard in https://github.com/rladies/rladies.github.io/pull/425
